src/topology/algebra/continuous_functions.lean

```diff
@@ -3,12 +3,32 @@ Copyright (c) 2019 Scott Morrison. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Scott Morrison
 -/
-import topology.algebra.ring
+import topology.algebra.module
+
+/- TODO: change subtype of continuous functions into continuous bundled functions. -/
```

Comment:
Is your intention to do this before we merge? I think eventually it's the right course.

Reply:
Hopefully we'll have the `continuity` tactic soon, and then bundled continuous functions can use that as an `auto_param`.

Reply:
No, I intend to do that in another PR, after I will have generalized `topological_monoid` with `has_continuous_mul`, so that I do things in the most general case since the beginning. In any case, I plan to leave the subtype of continuous functions, both so not to break the library and because with continuous bundled functions one cannot talk about continuous functions being a substructure of all functions but only that they have that particular structure themselves, and I do not know if someone in the future wants to use this. I will just add continuous bundled functions at the end of this file in a future PR and if anybody will want to remove the subtype they can do it themselves in the future.